### PR TITLE
Add additional output to permadelete

### PR DESCRIPTION
**Pull request by willkg**
This makes it a little clearer that it did something and how long it took from the output.

### CODE REVIEW

File: bin/permadelete_crash_data.py [hunk 1]
```diff
@@ -77,10 +78,15 @@ def cmd_permadelete(ctx, crashidsfile):
...
         click.echo(f"Working on {crashid!r}")
+        start_time = datetime.datetime.now()
         # delete from storage
+        click.echo(">>> Deleting from storage ...")
         crashstorage.delete_crash(crashid)
         # delete from es
+        click.echo(">>> Deleting from Elasticsearch ...")
         es_crashstorage.delete_crash(crashid)
+        end_time = datetime.datetime.now()
+        click.echo(f">>> Time: {end_time - start_time}")
```
Comment (willkg) [inline bin/permadelete_crash_data.py:89]: I was on the fence about whether to adjust `delete_crash` so it indicated whether it deleted anything or not. If it didn't delete anything, then that would indicate to us that maybe we got the crash id wrong.

I was also on the fence about whether to hit the `CrashVerify` endpoint and have it spit out the status of the crash report's data. That seems like a good check to do.

@relud What do you think?

Comment (relud) [reply bin/permadelete_crash_data.py:89]: those both seem like good things to have in a crash deletion report

**Review by relud — APPROVED** [review]

**Comment by willkg** [comment]
^^^ That adds a before/after call to `CrashVerify` so we know more about what actually happened. Now we get output like this:

```
app@socorro:/app$ ./bin/permadelete_crash_data.py local crashids.txt 
Working on 'a309b47f-a072-4d16-9138-de45c0240821'
>>> before: {'uuid': 'a309b47f-a072-4d16-9138-de45c0240821', 'raw_crash': False, 'processed_crash': False, 'elasticsearch_crash': False, 'telemetry_crash': False}
>>> Deleting from storage ...
>>> Deleting from Elasticsearch ...
>>> after: {'uuid': 'a309b47f-a072-4d16-9138-de45c0240821', 'raw_crash': False, 'processed_crash': False, 'elasticsearch_crash': False, 'telemetry_crash': False}
>>> Time: 0:00:00.192091

Working on '65af2601-8ea0-448a-9612-103ed0240821'
>>> before: {'uuid': '65af2601-8ea0-448a-9612-103ed0240821', 'raw_crash': False, 'processed_crash': False, 'elasticsearch_crash': False, 'telemetry_crash': False}
>>> Deleting from storage ...
>>> Deleting from Elasticsearch ...
>>> after: {'uuid': '65af2601-8ea0-448a-9612-103ed0240821', 'raw_crash': False, 'processed_crash': False, 'elasticsearch_crash': False, 'telemetry_crash': False}
>>> Time: 0:00:00.272631

Done!
```

Honestly, I'm on the fence about whether I like that or not. Part of me thinks we've got the crashstorage items so maybe we should use that to figure out the before/after without using `CrashVerify`. I think I may tinker with that tomorrow.

**Review by relud — APPROVED** [review]
this lgtm, i quite like the crash verify before/after

**Comment by willkg** [comment]
This sort of feels like bikeshedding, but I arrived at something that gives us a good understanding of what data we have before and after and does it in a way that is convenient for running the script in multiple environments without crossing streams.

```shell
app@socorro:/app$ echo 3dd11b5a-d5db-49f6-94f2-8e90a0240829 | ./bin/permadelete_crash_data.py /dev/stdin
Working on '3dd11b5a-d5db-49f6-94f2-8e90a0240829'
>>> before: ['gcs_raw_crash', 'gcs_processed_crash', 'gcs_dump_names', 'gcs_dump_upload_file_minidump', 'es_processed_crash']
>>> Deleting from storage ...
>>> Deleting from Elasticsearch ...
>>> after: []
>>> Time: 0:00:00.025893

Done!
app@socorro:/app$ echo 3dd11b5a-d5db-49f6-94f2-8e90a0240829 | ./bin/permadelete_crash_data.py /dev/stdin
Working on '3dd11b5a-d5db-49f6-94f2-8e90a0240829'
>>> before: []
>>> Deleting from storage ...
>>> Deleting from Elasticsearch ...
>>> after: []
>>> Time: 0:00:00.018158

Done!
```

@relud What do you think of that? ^^^

**Review by relud — APPROVED** [review]

**Comment by willkg** [comment]
Bah. Stupid non-static ordering mumbo jumbo.

**Comment by willkg** [comment]
Thank you!

**Pull request merged**